esp/services/esdl_svc_engine/esdl_binding.hpp

```diff
@@ -109,6 +109,7 @@ class EsdlServiceImpl : public CInterface, implements IEspService
     StringBuffer                m_serviceNameSpaceBase;
     StringAttr                  m_namespaceScheme;
     bool                        m_usesURLNameSpace;
+    bool                        m_returnSchemaLocationOnOK{false};
```

Comment:
Interesting I had to look up that syntax to see it was a valid alternative.
We usually use "bool m_returnSchemaLocationOnOK=false;"